spec/spec_helper.rb

```diff
@@ -6,7 +6,9 @@
     config.formatter = :documentation
 end
 
-require 'pry'
```

Comment:
there was a merge conflict here that I was able to resolve, nothing major, but was able to just rearrange and make it a little more organized at the same time.